grip/github_renderer.py

```diff
@@ -2,18 +2,18 @@
 from flask import abort, json
 
 
-def render_content(text, gfm=False, context=None,
+def render_content(text, api_url, gfm=False, context=None,
                    username=None, password=None):
     """Renders the specified markup using the GitHub API."""
     if gfm:
-        url = 'https://api.github.com/markdown'
```

Comment:
Nice! It's probably good that this is a setting instead of being hard-coded. Even better now that you can set this locally and never have to worry about passing it in the CLI each time.